olca-app/src/org/openlca/app/preferences/Preferences.java

```diff
@@ -8,18 +8,23 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static org.openlca.app.tools.graphics.themes.Themes.DARK;
+import static org.openlca.app.tools.graphics.themes.Themes.LIGHT;
+
 public final class Preferences extends AbstractPreferenceInitializer {
 
 	public static final String NUMBER_ACCURACY = "NUMBER_ACCURACY";
 	public static final String FORMAT_INPUT_VALUES = "FORMAT_INPUT_VALUES";
 	public static final String LAST_EXPORT_FOLDER = "LAST_EXPORT_FOLDER";
 	public static final String NO_INSTANT_SEARCH = "NO_INSTANT_SEARCH";
+	public static final String GRAPHICS_THEME = "GRAPHICS_THEME";
```

Comment:
GRAPHICS_THEME is too unspecific, something like GRAPHICAL_EDITOR_THEME would be better